provisioning/tasks/init-debian.yml

```diff
@@ -29,3 +29,7 @@
   set_fact:
     php_xhprof_html_dir: "/usr/share/php/xhprof_html"
   when: php_xhprof_html_dir is not defined
+
+- name: Install PHP PostgreSQL dependencies.
```

Comment:
Yeah, I'd like to follow the pattern of setting this up in a separate role. Will try to get to that soon.